src/components/SearchModal/ManageLists.tsx

```diff
@@ -27,28 +28,36 @@ const Wrapper = styled(Column)`
   height: 100%;
 `
 
-const RowWrapper = styled(Row)<{ active: boolean }>`
+const RowWrapper = styled(Row)<{ active: boolean; hasActiveTokens: boolean }>`
   background-color: ${({ active, theme }) => (active ? `${theme.colors.success}19` : 'transparent')};
   border: solid 1px;
   border-color: ${({ active, theme }) => (active ? theme.colors.success : theme.colors.tertiary)};
   transition: 200ms;
   align-items: center;
   padding: 1rem;
   border-radius: 20px;
+  opacity: ${({ hasActiveTokens }) => (hasActiveTokens ? 1 : 0.4)};
 `
 
 function listUrlRowHTMLId(listUrl: string) {
   return `list-row-${listUrl.replace(/\./g, '-')}`
 }
 
 const ListRow = memo(function ListRow({ listUrl }: { listUrl: string }) {
+  const { chainId } = useWeb3React()
```

Comment:
`useActiveWeb3React`, chainId will be undefined when user is not active

Reply:
@chef-jojo  done